castai/resource_node_configuration.go

```diff
@@ -161,9 +161,21 @@ func resourceNodeConfiguration() *schema.Resource {
 						"key_pair_id": {
 							Type:             schema.TypeString,
 							Optional:         true,
-							Description:      "AWS key pair ID to be used for provisioned nodes. Has priority over ssh_public_key",
+							Description:      "AWS key pair ID to be used for CAST provisioned nodes. Has priority over ssh_public_key",
 							ValidateDiagFunc: castval.ValidKeyPairFormat(),
 						},
+						"volume_type": {
```

Comment:
Would be nice to also add include in the unit test those new attributes -> TestAccResourceNodeConfiguration_basic

Reply:
Thanks, good point. Included as suggested.